src/handlers/commandHandler.ts

```diff
@@ -0,0 +1,59 @@
+import { Client, REST, Routes } from 'discord.js';
+import { readdir } from 'fs/promises';
+import { join } from 'path';
+import env from "../env";
+import type { APIApplicationCommand } from 'discord-api-types/v10';
+
+export async function loadCommands(client: Client) {
+    const commandsPath = join(process.cwd(), 'src', 'commands');
+    const commandFiles = await readdir(commandsPath);
+    const commands: any[] = [];
+
+    for (const file of commandFiles) {
+        if (!file.endsWith('.ts')) continue;
+
+        const command = await import(join(commandsPath, file));
+        if ('data' in command && 'execute' in command) {
+            client.commands.set(command.data.name, command);
+            commands.push(command.data.toJSON());
+            console.log(`Loaded command: ${command.data.name}`);
+        }
+    }
+
+    return commands;
+}
+
+export async function registerCommands(commands: any[]) {
+    const discordToken = env.DISCORD_TOKEN;
+    const clientId = env.CLIENT_ID;
+
+    if (!discordToken || !clientId) {
```

Comment:
```suggestion
    if (!discordToken || !clientId) {
```

Environment variable validatie hoort te gebeuren in de env.ts.